CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 ### Changed
+- New network setting N3H_LOG_LEVEL
```

Comment:
```suggestion
- New network setting via environment variable HC_N3H_LOG_LEVEL
```

Reply:
So let me get this straight, N3H_LOG_LEVEL is actually for n3h, then there is a SEPARATE env var for HC that controls that ENV var?